product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -73,7 +73,8 @@ Prior to creating your cluster, make sure you have enough resources. Without eno
 1.  In the **Storage** section, select your volume type from the **Volume Type** list. 
     - For Azure, BigAnimal currently supports Azure Premium SSD storage types. In **Volume Properties**, select the type and amount of storage needed for your cluster. See [the Azure documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types#premium-ssds) for more information. 
 
-    - For AWS, BigAnimal currently supports General Purpose SSD (GP3) storage. In **Volume Properties**, select the disk size for your cluster. 
+    - For AWS, BigAnimal currently supports General Purpose SSD (GP3). In **Volume Properties**, select the disk size for your cluster and configure the IOPS. It can take upto a few hours to tune and resize the disks in your cluster.
+ 
```

Comment:
delete this extra blank line so that the subsequent steps in the ordered list are numbered correctly

Reply:
ok